instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Comment from './Comment';
+import './CommentSection.css';
+
+CommentSection.propTypes = {
+    postData: PropTypes.shape({
+        comments: PropTypes.array,
+        timestamp: PropTypes.string
+    })
+};
+
+class CommentSection extends React.Component {
+
+    state = {
+        comments: this.props.post.comments,
+        username: ""
+    };
+
+    componentDidMount() {
+        const id = this.props.post.imageUrl;
+        this.addNewComment();
+    }
+
+    addNewComment = event => {
+        event.preventDefault();
+        if (this.state.inputText) {
```

Comment:
You're using `this.state.inputText`, but you don't have inputText living on your state.